docs/ja/azure_aks/05_deploy_containers_to_turtlebot3.md

```diff
@@ -0,0 +1,2705 @@
+        Removing librtmp1:amd64 (2.4+20151223.gitfa8646d-1ubuntu0.1) ...
+        Removing libgnutls30:amd64 (3.4.10-4ubuntu1.4) ...
+        Removing libhogweed4:amd64 (3.2-1ubuntu0.16.04.1) ...
+        Removing libgmp10:amd64 (2:6.1.0+dfsg-2) ...
+        Removing libgssapi-krb5-2:amd64 (1.13.2+dfsg-5ubuntu2.1) ...
+        Purging configuration files for libgssapi-krb5-2:amd64 (1.13.2+dfsg-5ubuntu2.1) ...
+        Removing libheimbase1-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
+        Removing libidn11:amd64 (1.32-3ubuntu1.2) ...
+        Removing libkrb5-3:amd64 (1.13.2+dfsg-5ubuntu2.1) ...
+        Removing libk5crypto3:amd64 (1.13.2+dfsg-5ubuntu2.1) ...
+        Removing libkeyutils1:amd64 (1.5.9-8ubuntu1) ...
+        Removing libkrb5support0:amd64 (1.13.2+dfsg-5ubuntu2.1) ...
+        Removing libnettle6:amd64 (3.2-1ubuntu0.16.04.1) ...
+        Removing libp11-kit0:amd64 (0.23.2-5~ubuntu16.04.1) ...
+        Removing libwind0-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
+        Removing libroken18-heimdal:amd64 (1.7~git20150920+dfsg-4ubuntu1.16.04.1) ...
+        Removing libsasl2-2:amd64 (2.1.26.dfsg1-14ubuntu0.1) ...
+        Removing libsasl2-modules-db:amd64 (2.1.26.dfsg1-14ubuntu0.1) ...
+        Removing libtasn1-6:amd64 (4.7-3ubuntu0.16.04.3) ...
+        Removing perl-modules-5.22 (5.22.1-9ubuntu0.6) ...
+        Processing triggers for libc-bin (2.23-0ubuntu11) ...
+        Removing intermediate container 66619cb3d2dd
+        ---> f818658cd8d0
+        Successfully built f818658cd8d0
+        Successfully tagged abelacr.azurecr.io/roboticbase/fiware-ros-turtlebot3-bridge:0.2.0
+        fiware@FIWARE-PC:~/example-turtlebot3$
+        fiware@FIWARE-PC:~/example-turtlebot3$
+        ```
+
+1. ACRのログイン
+
+    ```
+    $ az acr login --name ${ACR_NAME}
+    ```
+
+    - 実行結果（例）
+
+        ```
+        Login Succeeded
+        WARNING! Your password will be stored unencrypted in /home/fiware/.docker/config.json.
+        Configure a credential helper to remove this warning. See
+        https://docs.docker.com/engine/reference/commandline/login/#credentials-store
+        ```
+
+1. fiware-ros-turtlebot3-bridgeのイメージ登録
+
+    ```
+    $ docker push ${REPOSITORY}/roboticbase/fiware-ros-turtlebot3-bridge:0.2.0
+    ```
+
+    - 実行結果（例）
+
+        ```
+        The push refers to repository [abelacr.azurecr.io/roboticbase/fiware-ros-turtlebot3-bridge]
+        907bfd21b038: Pushed
+        3ab1fe801172: Pushed
+        dacf0636941f: Pushed
+        c1bc9ba02df9: Pushed
+        68dda0c9a8cd: Mounted from roboticbase/ros-master
+        f67191ae09b8: Mounted from roboticbase/ros-master
+        b2fd8b4c3da7: Mounted from roboticbase/ros-master
+        0de2edf7bff4: Mounted from roboticbase/ros-master
+        0.2.0: digest: sha256:1c3918f512a7810a36e8a35d1bac2182ab58edd5fe2585391005af5a11dc14e6 size: 1984
+        ```
+
+1. ユーザ名とパスワードの設定
+
+  
```

Comment:
これは起動失敗したときのログっぽい。
`git clone`先を間違えていたころのものかな。

Reply:
様々なログが混在してしまっています。

1回目の起動: 2645行目～2674行目
1回目の終了: 2675行目～2679行目
2回目の起動: 2680行目～2710行目
2回目の終了: 2711行目～2715行目
3回目の起動: 2716行目～2744行目
3回目の終了: 2745行目～2750行目
curlによる接続テスト: 2751行目～2764行目
4回目の起動: 2765行目～2795行目
4回目の終了: 2796行目～2800行目
5回目の起動: 2801行目～2831行目
5回目の終了: 2832行目～2836行目
6回目の起動(とエラー): 2837行目～2867行目


今回必要なのは1回分の起動ログなので、「1回目の起動: 2645行目～2674行目」だけを使いましょう。

Reply:
下記の1回目のログのみ残します。

1. turtlebot3の起動【turtlebot3-pc】

    ```
    @minikube|bash-4.3$ roslaunch turtlebot3_fake turtlebot3_fake.launch
    ```

    - 実行結果（例）

        ```
        ... logging to /home/fiware/.ros/log/053b87ca-4cf4-11e9-893a-b86b23c71f8e/roslaunch-turtlebot3-pc-30774.log
        Checking log directory for disk usage. This may take awhile.
        Press Ctrl-C to interrupt
        Done checking log file disk usage. Usage is <1GB.

        started roslaunch server http://172.17.0.1:41152/

        SUMMARY
        ========

        PARAMETERS
        * /robot_description: <?xml version="1....
        * /robot_state_publisher/publish_frequency: 50.0
        * /rosdistro: kinetic
        * /rosversion: 1.12.14
        * /tb3_model: waffle

        NODES
        /
            robot_state_publisher (robot_state_publisher/robot_state_publisher)
            rviz (rviz/rviz)
            turtlebot3_fake_node (turtlebot3_fake/turtlebot3_fake_node)

        ROS_MASTER_URI=http://localhost:11311

        process[turtlebot3_fake_node-1]: started with pid [30794]
        process[robot_state_publisher-2]: started with pid [30795]
        process[rviz-3]: started with pid [30808]
        [rviz-3] process has finished cleanly
        log file: /home/fiware/.ros/log/053b87ca-4cf4-11e9-893a-b86b23c71f8e/rviz-3*.log
        ^C[robot_state_publisher-2] killing on exit
        [turtlebot3_fake_node-1] killing on exit
        shutting down processing monitor...
        ... shutting down processing monitor complete
        done
        ```

Reply:
> 1回目の起動: 2645行目～2674行目
> 1回目の終了: 2675行目～2679行目
> 今回必要なのは1回分の起動ログなので、「1回目の起動: 2645行目～2674行目」だけを使いましょう。

ここまで詳細に書いたのに、なんで2645行目～2679行目を使うかな……
終了ログは不要です。この時点で終了しちゃったら06手順が動かないじゃないですか。

Reply:
申し訳ございません、修正箇所を間違えました。
下記に再修正します。
```
... logging to /home/fiware/.ros/log/053b87ca-4cf4-11e9-893a-b86b23c71f8e/roslaunch-turtlebot3-pc-30774.log
Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

started roslaunch server http://172.17.0.1:41152/

SUMMARY
========

PARAMETERS
* /robot_description: <?xml version="1....
* /robot_state_publisher/publish_frequency: 50.0
* /rosdistro: kinetic
* /rosversion: 1.12.14
* /tb3_model: waffle

NODES
/
    robot_state_publisher (robot_state_publisher/robot_state_publisher)
    rviz (rviz/rviz)
    turtlebot3_fake_node (turtlebot3_fake/turtlebot3_fake_node)

ROS_MASTER_URI=http://localhost:11311

process[turtlebot3_fake_node-1]: started with pid [30794]
process[robot_state_publisher-2]: started with pid [30795]
process[rviz-3]: started with pid [30808]
[rviz-3] process has finished cleanly
log file: /home/fiware/.ros/log/053b87ca-4cf4-11e9-893a-b86b23c71f8e/rviz-3*.log
```